.github/actions/publish/upload_to_splunkbase.py

```diff
@@ -52,7 +52,9 @@ def get_release_notes(tarball: str, version: str) -> Optional[str]:
     with tarfile.open(tarball, "r") as tar:
         for name in tar.getnames():
             if filename in name:
-                return tar.extractfile(name).read()
+                release_notes_encoded = tar.extractfile(name).read()
+                release_notes = release_notes_encoded.decode().split("\n")
+                return "\n".join(release_notes[1:])  # Remove the **UNRELEASED** header
```

Comment:
I think a regex replace might be more robust? (I didn't test this, just wrote it on-the-fly in the comment)
```suggestion
                release_notes = tar.extractfile(name).read().decode()
                return re.sub(r"^\** *unreleased *\**\n$", "", release_notes, flags=re.IGNORECASE | re.MULTILINE)
```

Reply:
Updating with working version, tested locally